src/main.rs

```diff
@@ -103,6 +103,12 @@ async fn start() -> Result<(), Box<dyn std::error::Error>> {
         max_expiration,
     };
 
+    if env::password_hash_salt() == env::DEFAULT_PASSWORD_SALT {
+        tracing::info!(
```

Comment:
Lets maybe even raise to warning?

Reply:
On a second thought, changing the salt of a running instance invalidates all encrypted pastes.
Also it seems all encrypted pastes are encrypted with the password combined with a nonce; is a custom specified salt necessary?

Reply:
argon2 requires a salt so … maybe yes, maybe no. I am not a crypto expert to be honest.

Reply:
> On a second thought, changing the salt of a running instance invalidates all encrypted pastes.

Well, we could extend the log and warn that changing will also make decrypting existing encrypted pastes impossible.

Reply:
Dropped this change for now.